citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -334,6 +353,150 @@ <h4 class="panel-title">
+		var movedToCollectionKey = segments[6];
+		var itemsKeys = [];
+		ui.helper.find('ul').children().each(function(){
+			itemsKeys.push($(this).text());
+			});
+		var itemsMoveData ={};
+		itemsMoveData["itemIds"] = itemsKeys;
+		itemsMoveData["collectionId"] = movedToCollectionKey;
+		$.ajax({
+			type: "POST",
+			url: 'items/move?[(${_csrf.parameterName})]=[(${_csrf.token})]',
+			data: JSON.stringify(itemsMoveData),
+			contentType : "application/json",
+			dataType: "json",
+			success: function (moveItemsAsyncResponse){
+		       pollMovedItemsStatuses(moveItemsAsyncResponse, collectionName);
+			 },
+			 error: function (moveItemsAsyncResponse) {
+				   showNotification(" Error while moving citations!", "danger", 10000);
+			 }
+		});
+		$('.fa-chevron-circle-left').show();
+		var checkedBoxes = $('input[type="checkbox"]:checked');
+		checkedBoxes.each(function(){
+			$(this).prop('checked', false);
+			$(this).parents('tr').removeClass('selected');
+			});
+		}
+});
+});	
+
+function pollMovedItemsStatuses(response,collectionName){
+	$.ajax({
+	     type: "POST",
+	     url: "items/move/status?[(${_csrf.parameterName})]=[(${_csrf.token})]",
+	     contentType : "application/json",
+		 dataType: "json",
+	     data: response.taskID,
+	     success: function (moveItemsStatusResponse) {
+	     if(moveItemsStatusResponse.taskStatus === "COMPLETE"){
+	       	 if(moveItemsStatusResponse.response.successItems.length != 0){
+	 		    message = " "+moveItemsStatusResponse.response.successItems.length+ " Citations moved successfully to ' "+collectionName+" ' ! <br /> <br />"+ moveItemsStatusResponse.response.successItems;
+	 			showNotification(message, "success", 20000);
+	 		 }
+	 		 if(moveItemsStatusResponse.response.failedItems.length != 0){
+	 			message =  " "+ moveItemsStatusResponse.response.failedItems.length+ " Citations not moved to ' "+collectionName+" ' ! <br /> <br />"+ moveItemsStatusResponse.response.failedItems;
+	 		   	showNotification(message, "danger", 20000);
+	 		 }
+	 		 clearAsyncTask(moveItemsStatusResponse.taskID);	
+	     }else{
+	       	 setTimeout(pollMovedItemsStatuses(moveItemsStatusResponse,collectionName),1000);
+	     }
+	     },
+	     error: function (moveItemsStatusResponse) {
+	       	showNotification("Error while polling citations statuses!", "warning", 1000)
+	        },
+	    });
+}
+
+function clearAsyncTask(taskID){
+	$.ajax({
+		type: "POST",
+		url: 'items/move/cleartask?[(${_csrf.parameterName})]=[(${_csrf.token})]',
+		data: taskID,
+		contentType : "application/json",
+		dataType: "json",
+		success: function (taskRemovedResponse){
+		 },
+		 error: function (taskRemovedResponse) {
+		 }
+	});
+}
+
+function showNotification(message, status, time){
+	$.notify('<i class="fas fa-exclamation-circle"></i>'+message, {
+		type: status,
+		offset: {
+			 x: 50,
+			 y: 90
+		},
+		timer: time,
+		animate: {
+			 enter: 'animated fadeInRight',
+			 exit: 'animated fadeOutRight'
+		},
+	});
+}
+		
```

Comment:
indentation of all the js

Reply:
Actually the indentation in javascript code is less. I checked it in Eclipse. Github is adding/showing up that extra space.